recipes/jupyter_resource_usage_proxy/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "jupyter_resource_usage_proxy" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyter_resource_usage_proxy-{{ version }}.tar.gz
+  sha256: 5d474bd3c78271d6924c4c202da89da01a1a3c176027796513567f950c6e92ab
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Conda-Build configures these for us in the build. So think we can simplify like so

```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```